client/tests/integration/components/ui/answer-prompt-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | ui/answer-prompt', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<Ui::AnswerPrompt />`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      <Ui::AnswerPrompt>
+        template block text
+      </Ui::AnswerPrompt>
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
+});
```

Comment:
Plz assert something real

Reply:
Updated the test to assert that passing optional `beside` boolean does/doesn't make the component have a grid cell layout.